pmd-core/src/main/java/net/sourceforge/pmd/rules/RuleFactory.java

```diff
@@ -196,6 +197,7 @@ public Rule buildRule(Element ruleElement) {
             rule = builder.build();
         } catch (ClassNotFoundException | IllegalAccessException | InstantiationException e) {
             LOG.severe(e.getMessage());
+            LOG.severe(ExceptionUtils.getStackTrace(e));
```

Comment:
FYI, you can use `LOG.log(Level.SEVERE, "My message", e);` to do both at the same time.